Code/Yahoo.Yui.Compressor.Build.MsBuild/CompressorTaskEngine.cs

```diff
@@ -395,10 +396,18 @@ private CompressionType GetCompressionTypeFor(FileSpec file)
         }
 
         [SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
+        [SuppressMessage("ReSharper", "AssignNullToNotNullAttribute")]
         private bool SaveCompressedText(StringBuilder compressedText)
         {
             // Note: compressedText CAN be null or empty, so no check.
 
+            var outputDirectory = Path.GetDirectoryName(Path.GetFullPath(OutputFile));
+            if (!Directory.Exists(outputDirectory))
+            {
+                Log.LogMessage(string.Format("Creating missing directory {0}", outputDirectory));

```

Comment:
I don't think you need the word "missing" in there.  It is the output directory so say "Creating output directory" instead